Kudu.Core/SourceControl/Git/LibGit2SharpRepository.cs

```diff
@@ -225,7 +227,7 @@ public void FetchWithoutConflict(string remoteUrl, string branchName)
                     using (tracer.Step("LibGit2SharpRepository Fetch"))
                     {
                         // This will only retrieve the "master"
-                        repo.Network.Fetch(remote);
+                        LibGit2Sharp.Commands.Fetch(repo, remote.Name, remote.RefSpecs.Select(r => r.Specification), new FetchOptions(), "LibGit2SharpRepository Fetch");
```

Comment:
posted a question http://stackoverflow.com/questions/41642090/upgrading-libgit2sharp-from-0-21-0-176-to-0-23-1